x-pack/plugins/watcher/public/constants/time_units.js

```diff
@@ -5,22 +5,39 @@
  */
 
 import { TIME_UNITS as COMMON_TIME_UNITS } from 'plugins/watcher/../common/constants/time_units';
+import { i18n } from '@kbn/i18n';
 
 export const TIME_UNITS = {
   [COMMON_TIME_UNITS.SECOND]: {
-    labelPlural: 'seconds',
-    labelSingular: 'second'
+    labelPlural: i18n.translate('xpack.watcher.constants.timeUnits.secondPluralLabel', {
```

Comment:
`xpack.watcher.constants.timeUnits.` => `xpack.watcher.timeUnits.`

Please apply it for all ids in this file.